buildSrc/src/main/resources/minimumGradleVersion

```diff
@@ -1 +1 @@
-7.4.1
+8.0
```

Comment:
@dbwiddis please revert this change, here is the context. We publish a lot Gradle plugins along with OpenSearch core, these plugins are used to run test, publications, and also scaffold OpenSearch plugins. By bumping this minimum version, we basically force everyone to make a switch to Gradle 8, example below for our own `security` plugin:

```
A problem occurred evaluating root project 'security'.
> Failed to apply plugin class 'org.opensearch.gradle.info.GlobalBuildInfoPlugin'.
   > Gradle 8.0+ is required
```

Reply:
ACK.  Do you want it back to 7.4.1 or 7.6 (required for JDK 19+)?

Reply:
Thanks,  7.4.1 is preferable (we are maintaining this compatibility), gives more freedom to plugin developers.